content/hi/docs/setup/best-practices/cluster-large.md

```diff
@@ -0,0 +1,88 @@
+---
+title: बड़े क्लस्टर्ज़ के लिए विचार
+weight: 20
+---
+
+एक क्लस्टर {{< glossary_tooltip text="नोड्स" term_id="node" >}} (भौतिक या वर्चुअल मशीन) का एक सेट है जो कुबेरनेट्स एजेंटों को चला रहा है, जिसे {{< glossary_tooltip text="कण्ट्रोल प्लेन" term_id="control-plane" >}} द्वारा प्रबंधित किया जाता है|
+कुबेरनेट्स {{< param "version" >}} 5000 नोड्स तक क्लस्टर का समर्थन करता है| अधिक विशेष रूप से, कुबेरनेट्स को ऐसे कॉन्फ़िगरेशन को समायोजित करने के लिए डिज़ाइन किया गया है जो निम्नलिखित *सभी* मानदंडों को पूरा करते हैं: 
+
+* प्रति नोड 110 पॉड से अधिक नहीं
+* 5000 से अधिक नोड्स नहीं
+* कुल पॉड्स 150000 से अधिक नहीं
+* 300000 से अधिक कुल कंटेनर नहीं 
+
+आप नोड्स जोड़कर या हटाकर अपने क्लस्टर को स्केल कर सकते हैं। आपके ऐसा करने का तरीका इस बात पर निर्भर करता है कि आपका क्लस्टर कैसे डिप्लॉय किया गया है|
+
+## क्लाउड प्रदाता संसाधन कोटा {#quota-issues}
```

Comment:
Tip:
```suggestion
## क्लाउड प्रदाता संसाधन कोटा
```
You don't have to copy over English-language custom fragment identifiers, unless you know there's a very good reason to.